frontend/fingertips-frontend/components/organisms/LineChartTable/index.tsx

```diff
@@ -163,75 +138,128 @@ const getCellHeader = (heading: string, index: number): ReactNode => {
-                <StyledAreaNameHeader>{data.areaName}</StyledAreaNameHeader>
-                <Table.CellHeader colSpan={2}></Table.CellHeader>
-                <StyledGreyHeader data-testid="england-header">
-                  Benchmark: <br /> England
-                </StyledGreyHeader>
-              </Table.Row>
-              <Table.Row>
-                <Table.CellHeader colSpan={5}></Table.CellHeader>
-                <StyledConfidenceLimitsHeader>
-                  95% confidence limits
-                </StyledConfidenceLimitsHeader>
-                <StyledGreyHeader></StyledGreyHeader>
-              </Table.Row>
-              <Table.Row>
-                {Object.values(LineChartTableHeadingEnum).map(
-                  (heading, index) => getCellHeader(heading, index)
-                )}
-              </Table.Row>
-            </>
-          }
-        >
-          {dataSortedByPeriod.map((point, index) => (
-            <Table.Row key={`${data.areaCode}-${point.period}--${index}`}>
-              <StyledAlignLeftTableCell numeric>
-                {point.period}
-              </StyledAlignLeftTableCell>
-              <StyledAlignLeftTableCell></StyledAlignLeftTableCell>
-              <StyledAlignRightTableCell numeric>
-                {point.count}
-              </StyledAlignRightTableCell>
-              <StyledAlignRightTableCell numeric>
-                {convertToPercentage(point.value)}
-              </StyledAlignRightTableCell>
-              <StyledAlignRightTableCell numeric>
-                {convertToPercentage(point.lower)}
-              </StyledAlignRightTableCell>
-              <StyledAlignRightTableCell numeric>
-                {convertToPercentage(point.upper)}
-              </StyledAlignRightTableCell>
-              <StyledBenchmarkValueTableCell data-testid="grey-table-cell">
-                {englandRowData.length
-                  ? convertToPercentage(englandRowData[index].value)
-                  : '-'}
-              </StyledBenchmarkValueTableCell>
+    <StyledDiv data-testid="lineChartTable-component">
+      <StyledTable
+        head={
+          <>
+            <Table.Row>
+              {data.map((area, index) => (
+                <React.Fragment key={area.areaName + index}>
+                  <StyledTitleRow colSpan={6}>
+                    {`${area.areaName} recent trend:`}
+                  </StyledTitleRow>
+                </React.Fragment>
+              ))}
+            </Table.Row>
+            <Table.Row>
+              {data.map((area, index) => (
+                <React.Fragment key={area.areaName}>
+                  <Table.CellHeader
+                    colSpan={getAreaNameCellSpan(index)}
+                  ></Table.CellHeader>
+                  <StyledAreaNameHeader>{area.areaName}</StyledAreaNameHeader>
+                  <Table.CellHeader colSpan={2}></Table.CellHeader>
+                </React.Fragment>
+              ))}
```

Comment:
This results in longer area names displaying strangely:
![Screenshot 2025-02-11 at 16 25 33](https://github.com/user-attachments/assets/82c57df3-cfe0-4afd-9f72-71cd886cd2ef)
I appreciate the designs _do_ look like this, but Leonie has raised the point that it doesn't look right. I'm double-checking that we definitely want to deviate from the designs, but I think it makes sense.

I think if we were to change this to something like the following it would allow us to have longer area names display nicely:
```suggestion
              {data.map((area, index) => (
                <React.Fragment key={area.areaName}>
                  {index === 0 ? <Table.CellHeader /> : null}
                  <StyledAreaNameHeader colSpan={5}>
                    {area.areaName}
                  </StyledAreaNameHeader>
                </React.Fragment>
              ))}
```